medcat/cat.py

```diff
@@ -539,64 +538,67 @@ def _print_stats(self,
 
     def train(self,
```

Comment:
How would you resume from checkpoint but also change the `lr` - a standard use-case? `lr` is defined in the `config`.

Reply:
I think we need to clarify which values in the `config` can be overridden during training resumption. At present, hyperparameter tuning has not been either a promoted feature or clearly documented for CAT trainers.

Reply:
I would assume that all parameters from the config should be changeable

Reply:
Not sure if it was an assumption or a confirmation. If it is changeable then why the config and the cdb have to be saved/pickled together?

Reply:
Hmm... I can see on `load_model_pack`, the local config will be merged with the config derived from the loaded cdb . Does that mean users have no chance to change things like `lr`? Should the merging be done from the other way around?

Reply:
No

Reply:
Can you show the full code snippet for how users currently can do if they want to load the model pack, alter the value `lr` and call the `train` method?

Reply:
So the users cannot do that via the local config.py but they still need to look into it to know there is a `$linking.optim` they can tweak.

Reply:
exactly